packages/workflow/src/ErrorReporterProxy.ts

```diff
@@ -0,0 +1,28 @@
+export interface IReportingOptions {
+	tags?: Record<string, string>;
```

Comment:
I would have expected `string[]`.

Reply:
sentry expects a `Record<string, Primitive>`. so, I updated this to use the same.